.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+  "recommendations": [
+    "editorconfig.editorconfig",
+    "formulahendry.dotnet-test-explorer",
+    "ms-dotnettools.csharp",
+    "streetsidesoftware.code-spell-checker"
```

Comment:
I think it is more of a personal preference than coupled to this repo. I suggest removing it.

Reply:
Done.